deploy/camel-catalog.yaml

```diff
@@ -0,0 +1,2252 @@
+components:
```

Comment:
@lburgazzoli , for multiple schema components catalog looks like cartesian join between them. Is it expected behavior?

Reply:
At the moment it is generated getting things out of the camel catalog without any logic, let me fix it